faq/ompio.inc

```diff
@@ -158,60 +158,60 @@ can try changing this parameter to 4 (hybrid) algorithm.
 
 $q[] = "What are the main parameters of the fs framework and components?";
 
-$anchor[] = "fs-parametesrs";
+$anchor[] = "fs-parameters";
```

Comment:
I'm a little worried about fixing anchor typos because we have encouraged people to link to specific FAQ items through the anchors.  I.e., we could be breaking a random internet link by fixing a typo in an anchor name.

Would it be ok to just put a comment here acknowledging the typo, and specifically stating that we're not fixing it so that we don't break random 3rd-party links to this FAQ item?

Reply:
Sure. I've amended this PR to revert all the anchor names, and just add comments about preserving the typos instead.